app/views/events/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Event</h1>
+<hr/>
+<dl class="dl-horizontal">
+  <dt><strong>Type:</strong></dt>
+  <dd><%= @event.event_type.name %></dd>
+  <dt><strong>Created At:</strong></dt>
+  <dd><%= date_and_time @event.created_at %></dd>
+  <dt><strong>Is done:</strong></dt>
+  <dd><%= format_boolean @event.is_done %></dd>
+  <dt><strong>Description:</strong></dt>
+  <dd><%= @event.description %></dd>
+</dl>
+
+
+<%= link_to t('.back', :default => t("helpers.links.back")),
+          events_path, :class => 'btn btn-default' %>
+
+<% if can?(:update, @task) %>
+  <%= link_to t('.edit', :default => t("helpers.links.edit")),
+          edit_event_path(@event), :class => 'btn btn-primary' %>
+<% end %>
```

Comment:
Completely redo this view to match the show views found else where in Binder.  For example, mimic the tasks show view.